Nix/Types.hs

```diff
@@ -104,7 +104,9 @@ stripIndent xs = NString Indented . removePlainEmpty . mergePlain . unsplitLines
   ls = stripEmptyOpening $ splitLines xs
   ls' = map (dropSpaces minIndent) ls
 
-  minIndent = minimum . map (countSpaces . mergePlain) . stripEmptyLines $ ls
+  minIndent = case stripEmptyLines ls of
```

Comment:
This fixes an occasional `Prelude.minimum: empty list` error, which (I assume) occurs when a multiline string is empty.